wiki/en/Running-a-Server.md

```diff
@@ -203,34 +182,33 @@ When the Server is running, the operating system will show an icon in the system
 <figure><img src="{% include img/en-screenshots/server-active.inc %}" style="float:left; margin-right:10px;" loading="lazy" alt="Image of the Jamulus Server icon"></figure> The Server is occupied
 
 
+# Running an Unregistered Server
 
-# Running a Private Server
-
-It is highly recommended to test your server in **public mode first** so as to narrow down any subsequent problems in private mode.
+It is highly recommended to test your Server on a **Public Directory** first so as to narrow down any subsequent problems in unregistered mode.
 
-### Port forwarding
+### Port forwarding 
 
-Normally, people from outside your home network cannot see things inside it. So if you want to operate a Jamulus server at home, you need to open a hole in your router to let Jamulus clients connect to it.
+If you are running a Server on your home network, people from outside will not be able to see things inside it. So if you want to operate a Jamulus Server, you need to open a hole in your router to let Jamulus clients connect to it.
 
 **The default port for the current version of Jamulus is UDP (not TCP) port 22124.**
 
 The exact setup of port forwarding differs for every router. For help see [portforward.com](https://portforward.com).
 
-Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your server (_but see also the note on dynamic DNS below_). You yourself have to connect to your own computer instead, since your computer runs the server. Therefore **only you** need to connect to `127.0.0.1`.
+Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself have to connect to your own computer instead, since your computer runs the Server. Therefore **only you** need to connect to `127.0.0.1`.
```

Comment:
> You yourself have to connect to your own computer instead, since your computer runs the Server. Therefore **only you** need to connect to `127.0.0.1`.

I don't like this.  It might be a fairly common case but some people will have multiple machines on a LAN at home (we've got loads...).  Saying "You need to connect to 127.0.0.1" isn't helpful - it needs to say "You'll use the _local_ address of the machine."  Unfortunately, I can't suggest a good, easy fix.

(`ipconfig` on Windows.  Linux seems to be in the process of getting rid of the legacy, well known network commands and I have no idea what the right one is now but it used to be `ifconfig` - which probably works on MacOS.  `hostname` works on both Windows and Linux to get the machine host name, which _might_ work, depending on the DNS set up of each machine on the LAN and the router...)

Reply:
I think we can't really do much to teach people stuff that's not directly related to Jamulus. How about:

"You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a client on the same machine as your Server, that would be `127.0.0.1`. "

Reply:
Or maybe state "localhost" (IMHO better than the ip directly)

Reply:
I don't think Windows clients like `localhost` - at least on my machine it never works.